03-transactions.md

```diff
@@ -120,6 +120,32 @@ If a revoked commitment transaction is published, the other party can spend this
 
     <revocation_sig> 1
 
+##### Leased channel (`option_will_fund`)
+
+If a `lease` applies to the channel, the `to_local` output of the `accepter`
+ensures the `leasor` funds are not spendable until the lease expires.
+
+In a leased channel, the `to_local` output that pays the `accepter` node
+is modified so that its CSV is equal to the greater of the
+`to_self_delay` or the `lease_end` - `blockheight`.
+
+    OP_IF
+        # Penalty transaction
+        <revocationpubkey>
+    OP_ELSE
+        MAX(`to_self_delay`, `lease_end` - `blockheight`)
+        OP_CHECKSEQUENCEVERIFY
+        OP_DROP
+        <local_delayedpubkey>
+    OP_ENDIF
+    OP_CHECKSIG
+
+The output is spent by an input with `nSequence` field set to
+MAX(`to_self_delay`, `lease_end` - `blockheight`)
+(which can only be valid after that duration has passed) and witness:
+
+    <local_delayedsig> <>
+
 #### `to_remote` Output
 
 If `option_anchor_outputs` applies to the commitment transaction, the `to_remote` output is encumbered by a one block csv lock.
```

Comment:
We'll also need to add a lease term timelock to the `to_remote` output of the lessee.  Otherwise the lessor could goad the lessee into force-closing (e.g., by letting an HTLC time out) and get their funds back immediately.